django-backend/src/linked_commons/settings.py

```diff
@@ -0,0 +1,130 @@
+"""
+Django settings for linked_commons project.
+
+Generated by 'django-admin startproject' using Django 2.2.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'k7wv@iw4nn9e2zzctd_1z=h91tzkaax#%dj!+vdu7!x2bu)ja$'

```

Comment:
We should not be committing a secret key, please use environment variables or something like that.

Reply:
Migrated to SECRET_KEY to env variables.